ci/test.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 # This script is used to ensure that the go.mod file is up to date.
 
+# Compatibility for Go 1.22
```

Comment:
I have already generated mocks for aliases. Without gotypesalias=1, mocks are generated without aliases.
Example from [failed test](https://github.com/uber-go/mock/actions/runs/11018617569/job/30599452788):
```
< func (c *MockEarthHumanPopulationCall) Return(arg0 int) *MockEarthHumanPopulationCall {
---
> func (c *MockEarthHumanPopulationCall) Return(arg0 import_mode.HumansCount) *MockEarthHumanPopulationCall {
```

`HumansCount` here - alias for `int`. On 1.23 and 1.22 with `GODEBUG=gotypealias=1` mockgen wiil generate mocks with correct types. On native 1.22 it wil generate `int` instead of `import_mode.HumansCount`.

Tests on 1.22 and 1.23 have different behavior now due to aliases.

Reply:
Understood - sorry, I meant to say: can you add this brief explanation into the test script :)

Reply:
I have tested it now. And with go 1.22 in go mod it does not work with aliases as i expected.
```sh
$ go version                                                       
go version go1.23.1 darwin/arm64
$ cat go.mod | grep "go 1."
go 1.22
$ cd mockgen               
$ go build . && cp mockgen /Users/USERNAME/go/bin/mockgen # copy binary to PATH
$ cat internal/tests/import_mode/go.mod | grep "go 1." #checking go version in nested go module 
go 1.23
$ cd internal/tests/import_mode                       
$ cat interfaces.go| grep "AddHumans" 
        AddHumans(HumansCount) []Human
$ go generate ./...
$ cat mock/interfaces.go | grep "AddHumans(arg0" #must have HumansCount and Human instead of int and Primate
func (m *MockEarth) AddHumans(arg0 int) []import_mode.Primate {
func (mr *MockEarthMockRecorder) AddHumans(arg0 any) *MockEarthAddHumansCall {
```

In my case it works only after i changed go version in go.mod and rebuild binary.

Do you have any ideas how to support go 1.22 and alias types together?